intro-HPC/ch_introduction.tex

```diff
@@ -135,9 +159,12 @@ \section{What is the \hpc?}
 \fi
 \ifbrussel
 All the nodes in the \hpc run under the ``\operatingsystemSL'' operating
-system, which is a clone of ``\operatingsystemRHEL'', 
+system, which is a clone of ``\operatingsystemRHEL'',
 with \emph{cpuset} support.
 \fi
+\ifgent
+All the nodes in the \hpc run under the ``\operatingsystem'' operating system, version 6.
```

Comment:
better include the version in our value for `\operatingsystem`? There's little use in using a template otherwise...